frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/UploadFileComposer.js

```diff
@@ -0,0 +1,138 @@
+import React, {useEffect, useRef} from 'react'
+import { useTranslation } from 'react-i18next'
+import CardSectionHeader from '../atoms/CardSectionHeader'
+import LabelFieldPair from '../atoms/LabelFieldPair'
+import CardLabel from '../atoms/CardLabel'
+import CardLabelError from '../atoms/CardLabelError'
+import CitizenInfoLabel from '../atoms/CitizenInfoLabel'
+import Header from '../atoms/Header'
+import { Loader } from '../atoms/Loader'
+import MultiUploadWrapper from '../molecules/MultiUploadWrapper'
+import TextInput from '../atoms/TextInput'
+
+const UploadFileComposer = ({module, customClass, config, register, setuploadedstate, onChange, formData, errors, localePrefix}) => {
+  const { t } = useTranslation()
+  let finalDocumentData = []
+  
+  //fetch mdms config based on module name
+  const tenant = Digit.ULBService.getStateId();
+  const { isLoading, data } = Digit.Hooks.useCustomMDMS(
+      tenant,
+      "works",
+      [
+          {
```

Comment:
@prachirane-eGov  add this line to filter response from api
                        "filter": "[?(@.module=='Project')]"